### PR TITLE
Update Travis to Use Xenial

### PR DESCRIPTION
**Pull request by jcbantuelle**
I'm flying a bit blind here but I took a look at the Travis CI error and it seemed surprising and unrelated to the recent updates:

```
Installing APT Packages
0.35s$ sudo -E apt-get -yq --no-install-suggests --no-install-recommends $(travis_apt_get_options) install chefdk docker-ce
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package chefdk
apt-get.diagnostics
apt-get install failed
The command "sudo -E apt-get -yq --no-install-suggests --no-install-recommends $(travis_apt_get_options) install chefdk docker-ce" failed and exited with 100 during .
```

Given that it couldn't find `chefdk`, my best guess is that the problem is using such an old version of Ubuntu and the package lists being archived or something.

I'm not aware of a way to test whether this fixes the Travis issue or not, but I've updated it to use Xenial (which is actually in alignment with the version the chef tests run against locally) and switched it from Chef Current to Chef Stable since Chef doesn't make any guarantees about Current. I figured that'd be more dependable for testing but I can revert if desired.

I welcome any thoughts or feedback you have regarding this, hopefully my guesswork here is on the right track.

### CODE REVIEW

**Comment by nickcharlton** [comment]
Oh nice!

So it seems to be failing, but because of a Chef change: https://travis-ci.org/github/nickcharlton/dokku-cookbook/builds/667415378

I think we need to accept the license: https://docs.chef.io/chef_license_accept/#chef-workstation

**Comment by jcbantuelle** [comment]
That's really nice that it'll run on PRs, and in retrospect seems obviously desirable. Sorry I didn't notice that initially.

I've updated the installation to auto-accept the Chef License, so between that and the upgrade to the newer Ubuntu everything appears to be passing now. Please let me know if there's anything else I can do to help, and thanks again for all of your feedback and attention on this.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-      - chef-current-trusty
+      - chef-stable-xenial
     packages:
       - chefdk
       - docker-ce
 
 services: docker
 
-install: chef exec bundle
+install: CHEF_LICENSE="accept" chef exec bundle
```
Comment (nickcharlton) [inline .travis.yml]: Ah hah! Nice, this is how you do it. I did read the doc, hoping to provide something a bit more specific but I couldn't get through the density of it. Thanks!

**Comment by nickcharlton** [comment]
No worries! It _might_ have been broken, I had to mess about with it a bit to get to the result page and GitHub hides it a little, too.

I'll merge this now.

**Pull request merged**